src/Lookup/UnresolvedEntityRedirectException.php

```diff
@@ -23,9 +24,15 @@ class UnresolvedEntityRedirectException extends EntityLookupException {
 	/**
 	 * @param EntityId $entityId
 	 * @param EntityId $redirectTargetId The ID of the target Entity of the redirect
+	 * @param string|null $message
+	 * @param Exception|null $previous
 	 */
-	public function __construct( EntityId $entityId, EntityId $redirectTargetId ) {
-		parent::__construct( $entityId, 'Unresolved redirect to ' . $redirectTargetId->getSerialization() );
+	public function __construct( EntityId $entityId, EntityId $redirectTargetId, $message = null, Exception $previous = null ) {
+		parent::__construct(
+			$entityId,
+			$message !== null ? $message : 'Unresolved redirect to ' . $redirectTargetId->getSerialization(),
```

Comment:
I like the `$message ?: ...` shortcut in such cases. Yes, I know, the `'0'` edge case. But seriously, who uses "0" as a message?